Terminal.Gui/Views/CheckBox.cs

```diff
@@ -35,7 +39,32 @@ public CheckBox ()
 
     private void CheckBox_MouseClick (object? sender, MouseEventEventArgs e)
     {
-        e.Handled = AdvanceCheckState () == true;
+#if CHECKBOX_SUPPORTS_DOUBLE_CLICK_ACCEPT
+        CheckState savedCheckState = CheckedState;
+#endif
```

Comment:
This is compile time constant not configuration manager option?

Reply:
It's not actually implemented. And I don't actually think it's a good idea. I'll just remove it.